core/src/validator.rs

```diff
@@ -151,17 +151,20 @@ impl Validator {
     pub fn new(
         mut node: Node,
         keypair: &Arc<Keypair>,
+        node_keypair: &Arc<Keypair>,
```

Comment:
Could we rename this and associated references in other files to `validator_group_keypair`? I think it would be clearer